### PR TITLE
Rename Worktree class to WorkingTree

### PR DESCRIPTION
**Pull request by jcouball**
### Description

Rename the `Worktree` class to `WorkingTree` to be more consistent with Git documentation and the term's use in public.

See [the definition of Working Tree in the Git glossary](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefworkingtreeaworkingtree) or [search the web for Git working tree](http://www.google.com/search?q=git+working+tree).

### What is changed?
All code and documentation that referred to worktree was changed to working tree.

### What else was changed and why?
* Updated the description of the RubyGit module in the project's README.md and in lib/ruby_git.rb
* Added the SVG version of the RubyGit Class Diagram so people don't have to look at it in LucidChart

### CODE REVIEW

File: lib/ruby_git.rb [hunk 1]
```diff
@@ -55,78 +55,79 @@ class << self
     attr_accessor :logger
   end
 
-  # Create an empty Git repository under the root worktree `path`
+  # Create an empty Git repository under the root woring tree `path`
```
Comment (frankthrock) [inline lib/ruby_git.rb]: Spelling error.

File: README.md [hunk 2]
```diff
@@ -8,9 +8,10 @@
 [![Test Coverage](https://api.codeclimate.com/v1/badges/5403e4613b7518f70da7/test_coverage)](https://codeclimate.com/github/main-branch/ruby_git/test_coverage)
 [![Slack](https://img.shields.io/badge/slack-main--branch/ruby__git-yellow.svg?logo=slack)](https://main-branch.slack.com/archives/C01CHR7TMM2)
 
-RubyGit is an object-oriented wrapper for the `git` command line tool for working with Worktrees
-and Repositories. It tries to make more sense out of the Git command line. See the object model
-in [this Lucid chart diagram](https://app.lucidchart.com/invitations/accept/7df13bab-3383-4683-8cb4-e76d539de93d)
+RubyGit is an object-oriented wrapper for the `git` command line tool for working git Repositories.
```
Comment (frankthrock) [inline README.md]: Perhaps you want to add 'with' here?  ...working WITH git repositories.  (Up to you...)

**Comment by jcouball** [comment]
Thanks, @frankthrock for the review.  I have fixed the issues you pointed out and added the following:

* Updated the description of the RubyGit module in the README.md
* Updated the description of the RubyGit module in the documentation in lib/ruby_git.rb
* Added the SVG for the RubyGit class diagram so people don't have to view it in Lucid Charts.  I updated the link in the README.md to point to this SVG instead of the Lucid Charts version.

**Review by frankthrock — APPROVED** [review]
Looks really good.  I like the verbose changes.

**Pull request merged**